### DISCUSSION TITLE
Putty software shows error

### DISCUSSION THREAD

**Discussion by Anjumvj** [learndevopsonline/learndevopsonline#1556] [Q&A]
Every time when i open putty software it shows error.

please check .Thanks
![Screenshot 2023-11-01 133229](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/21e73a55-8d82-4bcf-b8d4-3b38f38ae70c)
![Screenshot 2023-11-01 133422](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/e243c691-9bae-4688-8de2-2cbf3f42bc70)

**Comment by sreedharm07** [comment]
Hi @Anjumvj
attach allow all security group to instance

**Reply by Anjumvj** [reply]
already i choose all allow from security group

**Reply by sreedharm07** [reply]
can you check your security group inbound and outbound rules like this and attach screen shot here
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/c1f20db7-2bac-4720-9052-5de64e86ea9c)

**Reply by Anjumvj** [reply]
![Screenshot 2023-11-01 160515](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/316110ec-cc8e-49e5-bb28-d5c5336db3f4)

**Reply by sreedharm07** [reply]
send screen shot like above which i attached
in left side you can find security groups and open that and open your allow all security group and give screen shot here

**Reply by Anjumvj** [reply]
![Screenshot 2023-11-01 160752](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/047dd79e-746c-4926-9d31-6906cd3e7095)

**Reply by sreedharm07** [reply]
screen shot which you given now was attached with allow all security group and its able to log in **54.83.146.62** please try it

**Reply by Anjumvj** [reply]
![Screenshot 2023-11-01 161639](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/e5279d52-fc9e-4a96-8148-6b16da1b006a)
![Screenshot 2023-11-01 161647](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/4f69fe5a-97e8-475a-9484-520e1b27ec04)

**Reply by Anjumvj** [reply]
![Screenshot 2023-11-01 161835](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/7afb909a-210a-48bb-9a37-6e4e9e2929c2)

**Reply by sreedharm07** [reply]
are you using company laptop ?

**Reply by Anjumvj** [reply]
no. my own. using office wifi

**Reply by sreedharm07** [reply]
selecting this ?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/c1d1892c-52e0-4b74-8698-7443294d5dac)

**Reply by Anjumvj** [reply]
tried same issue facing

**Reply by sreedharm07** [reply]
its working for me but not for you means may be try once by connecting with another wifi network.

**Reply by Anjumvj** [reply]
ok let me try with another network and will update u

**Reply by Anjumvj** [reply]
yes solved. I tried with another wifi. Thanks a lot

**Reply by Anjumvj** [reply]
password is DevOps321 right. shows access denied

**Reply by sreedharm07** [reply]
Uname - centos
Password - DevOps321

**Reply by Anjumvj** [reply]
done.Thanks
